<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dario Messina</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="vm_tabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfoSu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionInfo_su"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAvviaAll"/>
    <addaction name="actionInterrompiAll"/>
    <addaction name="separator"/>
    <addaction name="actionNuova"/>
    <addaction name="actionImportMachines"/>
    <addaction name="actionExportMachines"/>
    <addaction name="separator"/>
    <addaction name="actionAbilitaAll"/>
    <addaction name="actionDisabilitaAll"/>
    <addaction name="separator"/>
    <addaction name="actionMostraRiepilogo"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMacchina">
    <property name="title">
     <string>&amp;Macchina</string>
    </property>
    <addaction name="actionRinomina"/>
    <addaction name="actionClona"/>
    <addaction name="actionElimina"/>
    <addaction name="separator"/>
    <addaction name="actionAvvia"/>
    <addaction name="actionPausa"/>
    <addaction name="actionReset"/>
    <addaction name="actionInterrompi"/>
    <addaction name="separator"/>
    <addaction name="actionVMLoad"/>
    <addaction name="actionVMSave"/>
    <addaction name="actionVMSaveAs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMacchina"/>
   <addaction name="menuInfoSu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Avvio rapido</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolbarNuova"/>
   <addaction name="toolbarClona"/>
   <addaction name="toolbarElimina"/>
   <addaction name="separator"/>
   <addaction name="toolbarAvviaAll"/>
   <addaction name="toolbarInterrompiAll"/>
   <addaction name="separator"/>
   <addaction name="toolbarAvvia"/>
   <addaction name="toolbarPausa"/>
   <addaction name="toolbarReset"/>
   <addaction name="toolbarInterrompi"/>
  </widget>
  <action name="actionInfo_su">
   <property name="text">
    <string>Info su...</string>
   </property>
  </action>
  <action name="actionNuova">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/resources/SVG PINO/nuovo_black.svg</normaloff>:/new/resources/SVG PINO/nuovo_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuova VM...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Apri</string>
   </property>
   <property name="toolTip">
    <string>Apri</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Salva con &amp;nome...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/exit/resources/SVG PINO/chiudi_black.svg</normaloff>:/exit/resources/SVG PINO/chiudi_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Esci</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Salva</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClona">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/clone/resources/SVG PINO/clona_black.svg</normaloff>:/clone/resources/SVG PINO/clona_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Clona VM...</string>
   </property>
   <property name="toolTip">
    <string>Clona macchina virtuale</string>
   </property>
  </action>
  <action name="actionImpostazioni">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/settings/resources/vm_settings_16px.png</normaloff>:/settings/resources/vm_settings_16px.png</iconset>
   </property>
   <property name="text">
    <string>Impo&amp;stazioni...</string>
   </property>
   <property name="toolTip">
    <string>Impostazioni</string>
   </property>
  </action>
  <action name="actionAvvia">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/start/resources/SVG PINO/avvia_black.svg</normaloff>:/start/resources/SVG PINO/avvia_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Avvia</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionInterrompi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/poweroff/resources/SVG PINO/interrompi_black.svg</normaloff>:/poweroff/resources/SVG PINO/interrompi_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Interrompi</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/reset/resources/SVG PINO/reset_black.svg</normaloff>:/reset/resources/SVG PINO/reset_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPausa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pause/resources/SVG PINO/pausa_black.svg</normaloff>:/pause/resources/SVG PINO/pausa_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Pausa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="toolbarAvvia">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/start/resources/SVG PINO/avvia_black.svg</normaloff>:/start/resources/SVG PINO/avvia_black.svg</iconset>
   </property>
   <property name="text">
    <string>Avvia</string>
   </property>
  </action>
  <action name="toolbarPausa">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/pause/resources/SVG PINO/pausa_black.svg</normaloff>:/pause/resources/SVG PINO/pausa_black.svg</iconset>
   </property>
   <property name="text">
    <string>Pausa</string>
   </property>
  </action>
  <action name="toolbarImpostazioni">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/settings/resources/vm_settings_32px.png</normaloff>:/settings/resources/vm_settings_32px.png</iconset>
   </property>
   <property name="text">
    <string>Impostazioni</string>
   </property>
   <property name="toolTip">
    <string>Impostazioni</string>
   </property>
  </action>
  <action name="toolbarClona">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/clone/resources/SVG PINO/clona_black.svg</normaloff>:/clone/resources/SVG PINO/clona_black.svg</iconset>
   </property>
   <property name="text">
    <string>Clona VM...</string>
   </property>
   <property name="toolTip">
    <string>Clona macchina virtuale</string>
   </property>
  </action>
  <action name="actionElimina">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/delete/resources/SVG PINO/elimina_black.svg</normaloff>:/delete/resources/SVG PINO/elimina_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Elimina VM</string>
   </property>
  </action>
  <action name="toolbarElimina">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/delete/resources/SVG PINO/elimina_black.svg</normaloff>:/delete/resources/SVG PINO/elimina_black.svg</iconset>
   </property>
   <property name="text">
    <string>Elimina VM</string>
   </property>
  </action>
  <action name="actionAvviaAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/start/resources/SVG PINO/avviatutte_black.svg</normaloff>:/start/resources/SVG PINO/avviatutte_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Avvia tutte</string>
   </property>
   <property name="toolTip">
    <string>Avvia tutte le macchine abilitate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="toolbarAvviaAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/start/resources/SVG PINO/avviatutte_black.svg</normaloff>:/start/resources/SVG PINO/avviatutte_black.svg</iconset>
   </property>
   <property name="text">
    <string>Avvia tutte</string>
   </property>
   <property name="toolTip">
    <string>Avvia tutte le macchine abilitate</string>
   </property>
  </action>
  <action name="actionInterrompiAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/poweroff/resources/SVG PINO/interrompitutte_black.svg</normaloff>:/poweroff/resources/SVG PINO/interrompitutte_black.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Interrompi tutte</string>
   </property>
   <property name="toolTip">
    <string>Interrompi tutte le macchine virtuali avviate</string>
   </property>
  </action>
  <action name="toolbarInterrompiAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/poweroff/resources/SVG PINO/interrompitutte_black.svg</normaloff>:/poweroff/resources/SVG PINO/interrompitutte_black.svg</iconset>
   </property>
   <property name="text">
    <string>Interrompi tutte</string>
   </property>
   <property name="toolTip">
    <string>Interrompi tutte le macchine virtuali avviate</string>
   </property>
  </action>
  <action name="toolbarInterrompi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/poweroff/resources/SVG PINO/interrompi_black.svg</normaloff>:/poweroff/resources/SVG PINO/interrompi_black.svg</iconset>
   </property>
   <property name="text">
    <string>Interrompi</string>
   </property>
   <property name="toolTip">
    <string>Interrompi macchina virtuale selezionata</string>
   </property>
  </action>
  <action name="toolbarReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/reset/resources/SVG PINO/reset_black.svg</normaloff>:/reset/resources/SVG PINO/reset_black.svg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Resetta macchina virtuale selezionata</string>
   </property>
  </action>
  <action name="toolbarNuova">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/resources/SVG PINO/nuovo_black.svg</normaloff>:/new/resources/SVG PINO/nuovo_black.svg</iconset>
   </property>
   <property name="text">
    <string>Nuova VM</string>
   </property>
   <property name="toolTip">
    <string>Crea macchina virtuale da golden copy</string>
   </property>
  </action>
  <action name="actionAbilitaAll">
   <property name="text">
    <string>Abilita tutte</string>
   </property>
  </action>
  <action name="actionDisabilitaAll">
   <property name="text">
    <string>Disabilita tutte</string>
   </property>
  </action>
  <action name="actionMostraRiepilogo">
   <property name="text">
    <string>Mostra riepilogo rete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVMSave">
   <property name="text">
    <string>Salva configurazione VM</string>
   </property>
  </action>
  <action name="actionVMLoad">
   <property name="text">
    <string>Carica configurazione VM...</string>
   </property>
  </action>
  <action name="actionVMSaveAs">
   <property name="text">
    <string>Salva configurazione VM come...</string>
   </property>
  </action>
  <action name="actionExportMachines">
   <property name="text">
    <string>Esporta macchine...</string>
   </property>
   <property name="toolTip">
    <string>Esporta macchine</string>
   </property>
  </action>
  <action name="actionImportMachines">
   <property name="text">
    <string>Importa macchine...</string>
   </property>
   <property name="toolTip">
    <string>Esporta macchine</string>
   </property>
  </action>
  <action name="actionRinomina">
   <property name="text">
    <string>Rinomina VM...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
